app/serializers/group_serializer.rb

```diff
@@ -57,4 +57,8 @@ def has_discussions
   def has_multiple_admins
     object.admins.count > 1
   end
+
+  def cover_photo
```

Comment:
I don't get it. cover_photo_* attributes are all on the group table already. So it does not look like a n+1 problem to me.